spring-rabbit/src/main/java/org/springframework/amqp/rabbit/log4j2/AmqpAppender.java

```diff
@@ -102,6 +102,11 @@
 	public static final String CATEGORY_LEVEL = "level";
 
 	/**
+	 * Key name for the thread name in the message properties.
+	 */
+	public static final String THREAD_HEADER_KEY = "thread";
```

Comment:
@Monofraps ,

sorry for such a long delay.

I would prefer to have it as simple `THREAD_NAME` name:
1. It is constant, so it is definitely already `KEY`
2. The usage in the code uniquely says that it is used for headers.

That's why I don't see reason to say too much in the constant if it doesn't bring much value or even overcomplicate it.

So, let me to fix it during merge!